### DISCUSSION TITLE
What is MRE, SAI and mk meaning?

### DISCUSSION THREAD

**Discussion by RodTDai** [lllyasviel/Fooocus#1792] [General]
Heyy guys!
What MRE, SAI and mk means on styles toggles? I can´t find a explanation anywhere.
Thanks

**Comment by poor7** [comment]
Abbreviation style developer.

**Comment by miraculix95** [comment]
poor7 gave a poor answer, I am afraid.

this is my understanding:
sai is stablity ai - predifined prompt extensions provided by Stability AI
mre - prompt extensions developed by the Fooocus-MRE project (some kind of advanced Fooocus, not maintained for 6 months afaik)
mk - prompt extensions sourced from the work by another person - https://github.com/K3nt3L/MK-Styles/tree/main - huge repo of styles

I hope this helps ... devs, please correct if false.

**Reply by mashb1t** [reply]
This is more detailed and correct.
